src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write your code here
+function createPromise(button) {
```

Comment:
create two separate functions for promises. you have js error `cannot read addEventListener of undefined`